core/validation/towns_fund/fund_specific_validation/resources/TF-grant-awarded.csv

```diff
@@ -1,4 +1,4 @@
-﻿Index Code,RDEL,CDEL,Total
+Index Code,RDEL,CDEL,Total
```

Comment:
I can't tell why/how this line has changed 👀  Even doing a byte-wise comparison they look identical to me.

Reply:
Nevermind - worked out why - it's got a unicode [Byte Order Mark (BOM)](https://en.wikipedia.org/wiki/Byte_order_mark) at the start of the file